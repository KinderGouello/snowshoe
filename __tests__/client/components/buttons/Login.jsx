import renderer from 'react-test-renderer';
import React from 'react';

import Login from '../../../../client/components/buttons/Login';

describe('Button - Login', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Login />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
