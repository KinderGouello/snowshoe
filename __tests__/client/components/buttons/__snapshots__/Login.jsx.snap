exports[`Button - Login should render correctly 1`] = `
<li>
  <a
    href="/auth/github">
    Log In
  </a>
</li>
`;
